xml/dhcp_vendor_class.xml

```diff
@@ -190,6 +190,18 @@
     <param pos="0" name="hw.device" value="Switch"/>
   </fingerprint>
 
+  <fingerprint pattern="^Aruba-Sensor(?:-([A-Z0-9]{6}))?$">
+    <description>HP Aruba UXI Sensor</description>
+    <example hw.model="Q9X65A">Aruba-Sensor-Q9X65A</example>
+    <example hw.model="R3R67A">Aruba-Sensor-R3R67A</example>
+    <example hw.model="R7H75A">Aruba-Sensor-R7H75A</example>
+    <example>Aruba-Sensor</example>
+    <param pos="0" name="hw.vendor" value="Aruba Networks"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.device" value="Sensor"/>
```

Comment:
Would "Network Management Device" be a better classification than "Sensor"? They are mentioned with the text "network management device" on CDW's website, but I was unable to locate such language on Aruba's website.

Reply:
I took that text from the [Aruba site](https://www.arubanetworks.com/products/network-management-operations/analytics-monitoring/user-experience-insight-sensors/). Other sites (resellers, presumably) have different descriptions - for instance, model Q9X65A is described in various places as:
* LTE Sensor
* LTE Sensor - wireless network management device
* LTE Wireless Sensor Network Management Device
* User Experience Insight LTE Sensor
* UXI sensor

The full description for the other 2 models mentioned in examples (R3R67A, R7H75A) seems to be _User Experience Insight Sensor G-Series - wireless network management device_.

So I think both `sensor` and `network management device` are reasonable descriptions. I don't have a strong opinion on the matter.

Reply:
To get another opinion, I contacted @TomSellers who agreed both are correct, but "Network Management Device" would be more specific.

Reply:
Updated `hw.device` to "Network Management Device"